app/services/order_cycle_form.rb

```diff
@@ -160,4 +183,41 @@ def removed_schedule_ids
   def new_schedule_ids
     @order_cycle.schedule_ids - existing_schedule_ids
   end
+
+  def can_update_selected_payment_methods?
+    @user.admin? || coordinator? || distributor?
+  end
+
+  def can_update_selected_shipping_methods?
+    @user.admin? || coordinator? || distributor?
+  end
+
+  def coordinator?
+    @user.enterprises.include?(@order_cycle.coordinator)
```

Comment:
Is this Ok ? or Does the user have to be the owner of the enterprise?

Reply:
I think this is right. Either the owner or any other users (aka "admins" or "managers") of the coordinator enterprise can be considered a "coordinator". 
FYI an owner is always a user thanks to `User#ensure_owner_is_manager`.